llama_index/langchain_helpers/text_splitter.py

```diff
@@ -416,12 +440,25 @@ class Split:
 
         # run postprocessing to remove blank spaces
         docs = self._postprocess_splits(docs)
+
+        self._callback_manager.on_event_end(
+            CBEventType.CHUNKING,
+            payload={"chunks": [x.text_chunk for x in docs]},
+            event_id=event_id,
+        )
         return docs
 
     def split_text(self, text: str, extra_info_str: Optional[str] = None) -> List[str]:
         """Split incoming text and return chunks."""
+        event_id = self._callback_manager.on_event_start(
```

Comment:
not for this PR: would be nice (just style wise) for these to be Python context managers (e.g. `with self._callback_manager.event(CBEventType.CHUNKING) as event: ...`)

Reply:
I can take a quick look at how to do this. Might be an easy change.

Reply:
Would be easy enough to do, lets do in another PR though